tools/pftdiff

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+while getopts ":h" option; do
+   case $option in
+      h) # display Help
+	   echo "script to compare two PFTs in a FATES parameter file.  takes three arguments: "
+	   echo "first argument is the parameter file name"
+	   echo "second argument is the first pft number (PFT numbering starts with 1)"
+	   echo "third argument is the second pft number (PFT numbering starts with 1)"
+         exit;;
+   esac
+done
+
+
+tempfile1=$(mktemp)
```

Comment:
@ckoven , is mktemp something the user should have set in their environment?  Do these four tempfiles just need some nominal name? It seems like they should have a different name though, right?

Reply:
ok, I see, mktemp is a system command.  I'm surprised thought that it works with this syntax. Have you printed out the temporary file names to make sure that they are unique and defined?

Reply:
yep.  they look something like `tmp.cGZMIIuLIH` and get put in the local directory.